client/webserver/site/src/js/state.ts

```diff
@@ -26,55 +37,68 @@ export default class State {
     return null
   }
 
-  /* dark sets the dark-mode cookie. */
-  static dark (dark: boolean) {
-    this.setCookie(darkModeCK, dark ? '1' : '0')
-    if (dark) {
-      document.body.classList.add('dark')
-    } else {
-      document.body.classList.remove('dark')
-    }
+  /*
+   * removeCookie tells the browser to stop using cookie. It's not enough to simply
+   * erase cookie value because browser will still send it to the server (with empty
+   * value), and that's not what server expects.
+   */
+  static removeCookie (cKey: string) {
+    document.cookie = `${cKey}=;expires=Thu, 01 Jan 1970 00:00:01 GMT;`
   }
 
   /*
    * isDark returns true if the dark-mode cookie is currently set to '1' = true.
    */
   static isDark () {
-    return document.cookie.split(';').filter((item) => item.includes(`${darkModeCK}=1`)).length
+    return document.cookie.split(';').filter((item) => item.includes(`${State.DarkModeCK}=1`)).length
   }
 
   /* passwordIsCached returns whether or not there is a cached password in the cookies. */
   static passwordIsCached () {
-    return !!this.getCookie(pwKeyCK)
+    return !!this.getCookie(State.PwKeyCK)
   }
 
-  /* store puts the key-value pair into Window.localStorage. */
-  static store (k: string, v: any) {
-    window.localStorage.setItem(k, JSON.stringify(v))
+  /* showLeftMarketDock returns whether or not user wants left market doc shown. */
+  static showLeftMarketDock () {
+    return this.fetchLocal(State.LeftMarketDockLK) === '1'
   }
 
-  /* clearAllStore remove all the key-value pair in Window.localStorage. */
-  static clearAllStore () {
-    window.localStorage.clear()
+  /*
+   * selectedAsset returns selected asset ID or null if none user hasn't selected one
+   * yet.
+   */
+  static selectedAsset (): number | null {
+    const assetIDStr = State.fetchLocal(State.SelectedAssetLK)
+    if (!assetIDStr) {
+      return null
+    }
+    return Number(assetIDStr)
```

Comment:
This one is also specific to **/wallets** and only used one place. Any reason to have this logic here instead of in **markets.ts**?

Reply:
> Any reason to have this logic here instead of in markets.ts?

Probably none, unlike `State.isDark` or `State.passwordIsCached` this method is unlikely to be re-used (used > 1 time), applied this suggestion.